src/tools/genpolicy/rules.rego

```diff
@@ -66,9 +64,65 @@ CreateContainerRequest:= resp {
     p_env_map := p_container.env_map  
     allow_env_map(p_env_map, i_env_map)
     
+    i_process := input.base.OCI.Process
+    
+    p_process := p_container.OCI.Process
+    
+    allow_args_by_equality(i_process, p_process, i_env_map)
+    
     print("PolicyCreateContainerRequest: true")
 }
 
+allow_args_by_equality(i_process, p_process, i_env_map) {
+    i_args := i_process.Args
+    p_args := p_process.OriginalArgs
```

Comment:
Should we:
- Rename the old Args to ProcessedArgs or DeprecatedArgs
- Rename OriginalArgs to Args

?

Perhaps we can remove ProcessedArgs / DeprecatedArgs in a future version of the Policy.

Reply:
split into deprecated commit